src/main/java/org/folio/circulation/domain/policy/lostitem/ChargeFee.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.circulation.domain.policy.lostitem;
+
+import java.math.BigDecimal;
+
+public abstract class ChargeFee {
```

Comment:
What does charge mean in this context? Could this be called a fee instead of a charge fee?

Reply:
What about `ItemFee` just to not mix up with FeeFine entity?

Reply:
Renamed to `ItemFee`